src/styles/index.css

```diff
@@ -0,0 +1,45 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+  background-color: rgb(86, 137, 228);
+}
+
+main {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+img:hover {
+  opacity: 0.5;
+}
+
+header {
+  width: 100%;
+  padding-bottom: 1rem;
+  margin-bottom: 0.5rem;
+  border-bottom: 1px solid #000;
+  background-color: rgb(86, 137, 228);
+}
+
+h2 {
+  margin-bottom: 1rem;
+}
```

Comment:
- [ ] Please use the correct `font family`, `font-size`, `font color`, and `font-weight` throughout your design by following the design given because it seems like there is a big difference between them. Try to look at the comparison below 👇 

**Given design**
![image](https://user-images.githubusercontent.com/90222110/196743583-97da7010-c4d7-4f7d-9a9f-6e93a1726805.png)

**Your Design**
![image](https://user-images.githubusercontent.com/90222110/196743952-bf2159d8-e961-457a-bb75-488be00a7fcd.png)
![image](https://user-images.githubusercontent.com/90222110/196744022-709cb598-d009-4dde-b81d-280be8fea7d7.png)